packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/components/offers/template.html

```diff
@@ -1,143 +1,64 @@
-                        data-translate="dedicatedCloud_offer_upgrade_full_backups"
-                        data-translate-values="{
-                            fullBackups: offer.getMinimumFullBackupsCount(),
-                        }"
-                    ></li>
-                    <li
-                        data-translate="dedicatedCloud_offer_upgrade_replication"
-                        data-ng-if="offer.hasReplication()"
-                    ></li>
-                    <li
-                        data-ng-if="offer.hasBackupDays()"
-                        data-translate="dedicatedCloud_offer_upgrade_choice_backup_days"
-                    ></li>
-                    <li
-                        data-ng-if="offer.hasScheduleHour()"
-                        data-translate="dedicatedCloud_offer_upgrade_choice_backup_time"
-                    ></li>
-                    <li
-                        data-ng-if="offer.isPremium()"
-                        data-translate="dedicatedCloud_offer_upgrade_long_term_retention"
-                    ></li>
-                    <li
-                        data-ng-if="offer.hasProxyPerHost()"
-                        data-translate="dedicatedCloud_offer_upgrade_proxy_backup"
-                    ></li>
-                </ul>
-            </oui-select-picker-section>
-            <oui-select-picker-footer>
-                <dedicated-price
-                    data-hide-tax="true"
-                    data-price="offer.getPrice()"
-                    data-tax="offer.getTax()"
-                    data-user="$ctrl.user"
-                ></dedicated-price>
-                <span data-translate="dedicatedCloud_offer_price_per_vm"></span>
-            </oui-select-picker-footer>
-        </oui-select-picker>
-    </div>
+    <oui-select-picker
+        class="col-md-4 mb-3"
+        data-ng-if="!$ctrl.previewMode"
+        data-ng-repeat="offer in $ctrl.backupOffers track by offer.offerName"
+        name="offer"
+        data-model="$ctrl.selectedOffer"
+        data-label="{{:: 'dedicatedCloud_offer_' + offer.offerName | translate }}"
+        data-variant="light"
+        data-on-change="$ctrl.onSelected(modelValue)"
+        data-values="[offer]"
+    >
+        <div data-ng-if="offer.offerName === $ctrl.actualOffer">
+            <oui-select-picker-description>
+                <span
+                    class="oui-status oui-status_success"
+                    data-translate="dedicatedCloud_offer_current_offer"
+                ></span>
+            </oui-select-picker-description>
+        </div>
+        <oui-select-picker-section>
+            <dedicated-cloud-datacenter-backup-offer-details
+                data-offer="offer"
+            ></dedicated-cloud-datacenter-backup-offer-details>
+        </oui-select-picker-section>
+        <oui-select-picker-footer>
+            <span data-translate="dedicatedCloud_offer_price_from"></span>
+            <dedicated-price
```

Comment:
We should use https://github.com/ovh/manager/pull/2214/files#diff-d6bd2bba6df155c89b3913e7a2d11453R1 when it's available, (not in this PR, it's just a note)